resources/protos/commit.proto

```diff
@@ -1,22 +1,33 @@
-syntax = "proto2";
+// Copyright 2017 Sourcerer, Inc. All Rights Reserved.
 
-package tutorial;
+syntax = "proto3";
+
+package app;
 
 option java_package = "app";
 option java_outer_classname = "CommitProtos";
 
-message Flag {
-    string type;
-    string value;
+// Types of information that can be extracted from commits.
+enum EnumLabelTypes {
+    LOC_ADDED = 0; // Lines of code added.
```

Comment:
Could you use two spaces before an inline comment? Pet peeve, but also if we start using Google code guides, it what they use across all languages.